.cci.jenkinsfile

```diff
@@ -5,22 +5,9 @@ parallel rpms: {
   def n = 5
   cosaPod(buildroot: true, runAsUser: 0, memory: "2Gi", cpu: "${n}") {
       checkout scm
-      shwrap("""
-        # fetch tags so `git describe` gives a nice NEVRA when building the RPM
-        git fetch origin --tags
-        git submodule update --init
-        ci/installdeps.sh
-        ci/install-extra-builddeps.sh
-        # Not in the path by default
-        export PATH="\$HOME/.cargo/bin:\$PATH"
-        # Our primary CI build goes via RPM rather than direct to binaries
-        # to better test that path, including our vendored spec file, etc.
-        # The RPM build expects pre-generated bindings, so do that now.
-        make -f Makefile.bindings bindings
-        cd packaging
-        RPM_BUILD_NCPUS=${n} make -f Makefile.dist-packaging rpm
-        mv \$(find . -name '*.rpm') ..
-      """)
+      shwrap("""./ci/coreosci-rpmbuild.sh
```

Comment:
CI is breaking because `${n}` isn't defined.

Hmm... let's just do `RPM_BUILD_NCPUS=${n} ./ci/coreosci-rpmbuild.sh` here? That way we don't have to pass `n` itself as a parameter.

Reply:
Yep, done!